src/redist/targets/packaging/windows/clisdk/bundle.wxs

```diff
@@ -14,6 +14,18 @@
         WixBundleInstalled OR ((NOT (DOTNETHOME_X64 ~= DOTNETHOME_X86)) OR DOTNETHOMESIMILARITYCHECKOVERRIDE)
     </bal:Condition>
       
+    <?if $(var.Platform)=x86?>
+      <!-- "Preview 1" 3.0.100-preview-009812-win-x86 upgrade-code -->
+      <RelatedBundle Action="Upgrade" Id="{489A47A1-5D69-51F1-3D54-3F8FE92E0963}"/>
+      <!-- "Preview 2" 3.0.100-preview-010184-win-x86 upgrade-code -->
+      <RelatedBundle Action="Upgrade" Id="{C1DD86C1-5FE4-573E-63FF-6BD0F23EBE69}"/>
+    <?elseif $(var.Platform)=x64?>
+      <!-- "Preview 1" 3.0.100-preview-009812-win-x64 upgrade-code -->
+      <RelatedBundle Action="Upgrade" Id="{18D28236-562F-73EB-340C-2B4B3A5347BB}"/>
+      <!-- "Preview 2" 3.0.100-preview-010184-win-x64 upgrade-code -->
+      <RelatedBundle Action="Upgrade" Id="{B47FC45A-5547-64E8-CFD1-B4C5CA275757}"/>
```

Comment:
Will this some day upgrade preview 1 and 2 to 3.0.200 if we don't remove this?

Reply:
In other words, does this make 3.0 preview 1 and 3.0 preview 2 appear to be in every "product band"?

Reply:
When you install a bundle that defines an upgrade relation ship, then any of those upgrade codes would automatically be removed once you install the current bundle. My understanding is that we'd want 3.0.100 to be a family and 3.0.200, but if we wanted to, we could add the 3.0.100 family of upgrade codes in the 3.0.200 bundles and it would remove those for us.

Reply:
You bring up a great question - I'll see if I can condition this in some way; since the upgrade code will be static for 3.0.1, maybe we can use that fact.